docs/en/policies.rst

```diff
@@ -131,3 +131,7 @@ Before hooks are expected to return one of three values:
 - ``false`` The user is not allowed to proceed with the action.
 - ``null`` The before hook did not make a decision, and the authorization method
   will be invoked.
+  
+Applying Policies
+-----------------
+https://book.cakephp.org/authorization/2/en/component.html#applying-policy-scopes
```

Comment:
```suggestion
See :ref:`applying-policy-scopes` for how to apply policies in your controller actions.
```

We should be able to use a `ref` directive to create a link.

Reply:
And shouldn't this rather be a note in the "Policy Scopes" section, given that this specifically links to applying scopes?

btw, everything after "Policy Result Objects" seems to be incorrectly nested.

Reply:
> btw, everything after "Policy Result Objects" seems to be incorrectly nested.

I can get that fixed outside of these changes.